mmv1/products/compute/api.yaml

```diff
@@ -15647,6 +15647,99 @@ objects:
+      guides:
+        'Official Documentation':
+          'https://cloud.google.com/load-balancing/docs/tcp/internal-proxy'
+      api: 'https://cloud.google.com/compute/docs/reference/rest/beta/targetTcpProxies'
+    min_version: beta
+    async: !ruby/object:Api::OpAsync
+      operation: !ruby/object:Api::OpAsync::Operation
+        kind: 'compute#operation'
+        path: 'name'
+        base_url: 'projects/{{project}}/regions/{{region}}/operations/{{op_id}}'
+        wait_ms: 1000
+      result: !ruby/object:Api::OpAsync::Result
+        path: 'targetLink'
+      status: !ruby/object:Api::OpAsync::Status
+        path: 'status'
+        complete: 'DONE'
+        allowed:
+          - 'PENDING'
+          - 'RUNNING'
+          - 'DONE'
+      error: !ruby/object:Api::OpAsync::Error
+        path: 'error/errors'
+        message: 'message'
+    parameters:
+      - !ruby/object:Api::Type::ResourceRef
+        name: 'region'
+        resource: 'Region'
+        imports: 'name'
+        required: true
+        input: true
+        description: |
+          The region where the regional proxy resides.
+    properties:
+      - !ruby/object:Api::Type::Time
+        name: 'creationTimestamp'
+        description: 'Creation timestamp in RFC3339 text format.'
+        output: true
+      - !ruby/object:Api::Type::String
+        name: 'description'
+        description: 'An optional description of this resource.'
+        input: true
+      - !ruby/object:Api::Type::Integer
+        name: 'id'
+        description: 'The unique identifier for the resource.'
+        output: true
+      - !ruby/object:Api::Type::String
+        name: 'name'
+        description: |
+          Name of the resource. Provided by the client when the resource is
+          created. The name must be 1-63 characters long, and comply with
+          RFC1035. Specifically, the name must be 1-63 characters long and match
+          the regular expression `[a-z]([-a-z0-9]*[a-z0-9])?` which means the
+          first character must be a lowercase letter, and all following
+          characters must be a dash, lowercase letter, or digit, except the last
+          character, which cannot be a dash.
+        input: true
+        required: true
+      - !ruby/object:Api::Type::Enum
+        name: 'proxyHeader'
+        description: |
+          Specifies the type of proxy header to append before sending data to
+          the backend.
+        values:
+          - :NONE
+          - :PROXY_V1
+        update_verb: :POST
+        update_url: 'projects/{{project}}/{{region}}/targetTcpProxies/{{name}}/setProxyHeader'
+      - !ruby/object:Api::Type::ResourceRef
+        name: 'service'
+        resource: 'RegionBackendService'
+        imports: 'selfLink'
+        description: |
+          A reference to the BackendService resource.
+        required: true
+        update_verb: :POST
+        update_url: 'projects/{{project}}/{{region}}/targetTcpProxies/{{name}}/setBackendService'
```

Comment:
This should probably be `'projects/{{project}}/regions/{{region}}/targetTcpProxies/{{name}}/setBackendService'`?

Reply:
On that note, can you write an update test that uses this? That would have caught this issue

Reply:
Can do - I suppose there isn't an easy way to do a generated one? All of the generated update tests seem to have some stuff for that under `tpgtools/`. Seems like the easiest way would be just to have a handwritten test.

Reply:
Unfortunately we can't do automated update tests right now. Handwritten is the way to go